schema.graphql

```diff
@@ -115,8 +115,8 @@ type Settlement @entity {
   id: ID!
   "Transaction hash"
   txHash: Bytes!
-  "First trade timestamp"
-  firstTradeTimestamp: Int!
```

Comment:
I thought we would use that in the UI.
Have we agreed to remove it?

Reply:
I'm pretty sure we agreed on removing it. But please let me know if you want to keep it.

Reply:
Discussed with Rami, Need to keep this prop